src/shared/install.c

```diff
@@ -1040,22 +1040,28 @@ static int find_symlinks_in_scope(
         return 0;
 }
 
-static void install_info_free(InstallInfo *i) {
+static void install_info_clear(InstallInfo *i) {
```

Comment:
more often than not we call this "weak destructors" `xyz_done()`. We probably should decide on a preferred naming and then sanction that in the coding style.

Reply:
But then again we didn't even add to the coding style that _new()/_free()/_ref()/_unref() is what you are supposed to call constructors/destructors/reference counting calls...

Reply:
`git grep` says that `_clear()` is used more often than `_done()`…